assets/styles/layout/_options.scss

```diff
@@ -65,6 +85,13 @@
 
   }
 
+  &__filter{
+    button{
+      border-bottom-left-radius: 0px!important;
+      border-bottom-right-radius: 0px!important;
```

Comment:
is the `!important` really necessary? Its always a bad idea to put that in css if it can be done a different way, e.g. by specifying the selector more precisely

Reply:
I didn't invent this; I copied the styling from the &__view button styling. Taking away !important means that the highlights underneath when you mouse over a filter would get shaped with a curve.

<img width="428" alt="Screenshot of buttons" src="https://github.com/MbinOrg/mbin/assets/8404538/7bf1988d-7bf1-436a-b2b1-e4e6977b34db">

Reply:
Got it :+1:

Reply:
The goal of this is to override the curved edges when there is more of the element to the left and right. So in the screenshot, that appears incorrect to me as it should be gaining 0px bottom left and bottom right, because it appears in the middle of that selection bar.

options on the far left edge would have 0px right only, the rightmost edge would have 0px left only, and everything in the middle gets 0px left and right.

you should be able to accomplish this with things like `:first-child` or `:not(:first-child)`, etc. I did this when rearranging the bar last time.

To try to give visual examples, the goal is that the curve highlight matches the curve of the bar itself

![image](https://github.com/MbinOrg/mbin/assets/146029455/f94bb237-4233-435c-bd87-91aac30b6c63)
![image](https://github.com/MbinOrg/mbin/assets/146029455/b55cb75c-5526-4a6b-a652-5522a4a022fe)
![image](https://github.com/MbinOrg/mbin/assets/146029455/97a7aa3c-21f3-4760-91a5-375f5e6a7d63)

(bottom one has the curve but very subtle)

Reply:
Yeah, sounds good - it's currently working and I think it's using the approach you described. The screenshot was just an illustrative example for why we need !important in it for the time being until some other approach gets figured out.